static_src/test/functional/pageobjects/user_role.element.js

```diff
@@ -9,5 +9,36 @@ import BaseElement from './base.element';
 // TODO attach to class as static property
 
 export default class UserRoleElement extends BaseElement {
+  // TODO: Look into why "this.element('#org_role' + guid).checked
+  // does not work. Currently, it doesn't work that way.
+  setAndGetUserRole(testUrl, cookieValue) {
+    this.setUserRole(testUrl, cookieValue);
+    return this.getUserRole(testUrl);
+  }
 
+  setUserRole(testUrl, cookieValue) {
+    this.browser.url(testUrl);
```

Comment:
navigating to a new page shouldn't be here in the element. (Unless the element actually would navigate to another page in real life)